pkg/controller/middleware/auth.go

```diff
@@ -53,8 +53,6 @@ func RequireAuth(cacher cache.Cacher, authProvider auth.Provider, db *database.D
 				// If it's been more than the TTL since we've seen this session,
 				// expire it by creating a new empty session.
 				if time.Since(t) > sessionIdleTTL {
-					authProvider.ClearSession(ctx, session)
```

Comment:
Clearing the session here actually makes it so the login controller "forgets" which session to revoke.